pkgs/development/compilers/llvm/git/compiler-rt/default.nix

```diff
@@ -72,8 +73,12 @@ stdenv.mkDerivation {
     # ld-wrapper dislikes `-rpath-link //nix/store`, so we normalize away the
     # extra `/`.
     ./normalize-var.patch
-  ] # Prevent a compilation error on darwin
-    ++ lib.optional stdenv.hostPlatform.isDarwin ./darwin-targetconditionals.patch
+  ]
+    ++ lib.optionals stdenv.hostPlatform.isDarwin [
+      ../../common/compiler-rt/darwin-plistbuddy-workaround.patch
+      # Prevent a compilation error on darwin
+      ./darwin-targetconditionals.patch
+    ]
```

Comment:
Can we make this unconditional?

Reply:
> The armv7l patch below this was specifically made conditional in [d616ae8](https://github.com/NixOS/nixpkgs/commit/d616ae8c6de5294cbd7db2431325f6b5387a4262)

That would no longer apply when targeting staging because we need to rebuild it anyway already.

Applying the patches always makes it easier to notice when they no longer apply when updating on other platforms.

Reply:
I rebased and made the patches non-optional. Verified v11, v12, v13, v14 on `aarch64-darwin` (with stdenv rebuild), and verified git at least evaluates. Also did a build of v11 on `x86_64-linux`.